docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -1,138 +1,81 @@
-
-This manifest includes the Kubernetes integration to collect Kubernetes metrics,
-System integration to collect system level metrics and logs from nodes, and
-the Pod's log collection using <<kubernetes-provider,dynamic inputs and Kubernetes provider>>.
 
 [discrete]
-== Settings
+== Step 2: Connect to the {stack}
 
 Set the {es} settings before deploying the manifest:
 
 [source,yaml]
 ------------------------------------------------
 - name: ES_USERNAME
-  value: "elastic"
+  value: "elastic" <1>
 - name: ES_PASSWORD
-  value: "passpassMyStr0ngP@ss"
+  value: "passpassMyStr0ngP@ss" <2>
 - name: ES_HOST
-  value: "https://somesuperhostiduuid.europe-west1.gcp.cloud.es.io:443"
+  value: "https://somesuperhostiduuid.europe-west1.gcp.cloud.es.io:443" <3>
 ------------------------------------------------
 
-// Begin collapsed section
-[%collapsible]
-.Configuration details
-====
-****
-
-[cols="2*<a"]
-|===
-| Setting | Description
-
-include::configuration/env/shared-env.asciidoc[tag=es-username]
-
-include::configuration/env/shared-env.asciidoc[tag=es-password]
-
-include::configuration/env/shared-env.asciidoc[tag=es-host]
-
-|===
+<1> The basic authentication username used to connect to {es}.
+<2> The basic authentication password used to connect to {kib}.
+<3> The {es} host to communicate with.
 
 Refer to <<agent-environment-variables>> for all available options.
 
-****
-====
-
-[discrete]
-=== Run {agent} on master nodes
 
-Kubernetes master nodes can use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
-to limit the workloads that can run on them. The manifest for standalone {agent} defines
-tolerations to run on master nodes. Agents running on master nodes collect metrics from the control plane
-components (scheduler, controller manager) of Kubernetes.
-To disable {agent} from running on master nodes, remove the following part of the DaemonSet spec:
 
-[source,yaml]
-------------------------------------------------
-spec:
- tolerations:
- - key: node-role.kubernetes.io/master
-   effect: NoSchedule
-------------------------------------------------
+[discrete]
+== Step 3: Configure tolerations
+include::run-container-common/agent-tolerations.asciidoc[]
 
+:manifest: elastic-agent-standalone-kubernetes.yaml
+[discrete]
+== Step 4: Deploy the {agent}
+include::run-container-common/deploy-elastic-agent.asciidoc[]
 
 [discrete]
-== Deploy
-To deploy to Kubernetes, run:
+== Step 5: View your data in {kib}
 
-["source", "sh", subs="attributes"]
-------------------------------------------------
-kubectl create -f elastic-agent-standalone-kubernetes.yaml
-------------------------------------------------
+. Launch {kib}:
++
+--
+include::{observability-docs-root}/docs/en/shared/open-kibana/widget.asciidoc[]
+--
 
-To check the status, run:
+. You can see data flowing in by going to *Analytics -> Discover* and selecting the index `+metrics-*+`, or even more specific, `+metrics-kubernetes.*+`.
```

Comment:
it is shown with `+` https://observability-docs_2366.docs-preview.app.elstc.co/guide/en/fleet/master/running-on-kubernetes-standalone.html#_step_5_view_your_data_in_kibana is it expected?